crates/bevy_asset/src/asset_server.rs

```diff
@@ -14,17 +14,29 @@ use parking_lot::{Mutex, RwLock};
 use std::{collections::hash_map::Entry, path::Path, sync::Arc};
 use thiserror::Error;
 
-/// Errors that occur while loading assets with an `AssetServer`
+/// Errors that occur while loading assets with an `AssetServer`.
 #[derive(Error, Debug)]
 pub enum AssetServerError {
+    /// Asset folder is not a directory.
     #[error("asset folder path is not a directory: {0}")]
     AssetFolderNotADirectory(String),
+
+    /// No asset loader was found for the specified extensions.
     #[error("no `AssetLoader` found{}", format_missing_asset_ext(.extensions))]
-    MissingAssetLoader { extensions: Vec<String> },
+    MissingAssetLoader {
+        #[allow(missing_docs)]
```

Comment:
why allowing missing docs here?

Reply:
It seemed redundant because of the line for MissingAssetLoader itself.

Reply:
@mockersf Should I put a short description then? `/// The list of extensions`, maybe. If so, is there a way to describe the tuple fields as well?

Reply:
Wait, were you asking if `#[allow(missing_docs)]` is needed? It is, because struct fields in enum variants are documentable and thus would influence the coverage :thinking:

Reply:
Oh okay, I skimmed through `get_path_asset_laoder` and didn't catch the "sub-extension" behavior, which is quite interesting actually as one could have a loader for, say, `config.js` but not `js` extensions. I should document that behavior in `load` as well.